install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/templates/control.j2

```diff
@@ -5,11 +5,14 @@ Maintainer: SecureDrop Team <securedrop@freedom.press>
 Homepage: https://securedrop.org
 Build-Depends: debhelper (>= 9), dh-python, python3-all, python3-setuptools, dh-systemd, dh-virtualenv
 Standards-Version: 3.9.8
-X-Python3-Version: >= 3.5
 
 Package: securedrop-app-code
 Architecture: amd64
 Conflicts: libapache2-mod-wsgi,supervisor
 Replaces: libapache2-mod-wsgi,supervisor
+{% if securedrop_build_focal_support %}
+Depends: ${dist:Depends}, ${misc:Depends}, ${python3:Depends}, apache2, apparmor-utils, coreutils, gnupg2, haveged, libapache2-mod-xsendfile, libpython3.8, paxctld, python3, redis-server, securedrop-config, securedrop-keyring, sqlite3
+{% else %}
 Depends: ${dist:Depends}, ${misc:Depends}, ${python3:Depends}, apache2, apparmor-utils, coreutils, gnupg2, haveged, libapache2-mod-xsendfile, libpython3.5, paxctld, python3 (>= 3.5), python3 (<< 3.6), redis-server, securedrop-config, securedrop-keyring, sqlite3
+{% endif %}
```

Comment:
Much improved! As discussed today in tech meeting, can we further consolidate the dependencies for `libpython3.8` -> `libpython3` for both platforms? Maybe that's not possible, but if it is, it'd greatly simplify the logic in play.

Reply:
It seems this can not be done.

```
# apt install libpython3  
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libpython3
```